blueman/main/NetConf.py

```diff
@@ -325,47 +331,47 @@ def get_default(cls):
             NetConf.default_inst = n
             return n
 
-    def __init__(self):
+    def __init__(self) -> None:
         logging.info("init")
         self.version = class_id
-        self.dhcp_handler = None
+        self.dhcp_handler: Optional[DHCPHandler] = None
         self.ipt_rules: List[Tuple[str, str, str]] = []
 
         self.ip4_changed = False
 
-    def set_ipv4(self, ip, netmask):
+    def set_ipv4(self, ip: Optional[str], netmask: Optional[str]) -> None:
```

Comment:
How are these optional, they shouldn't ever be afaik.

Reply:
See `Network._disabled_network` :confused:

Reply:
Hmm, ok.